src/Confluent.Kafka/Admin/DeleteAclsResult.cs

```diff
@@ -0,0 +1,110 @@
+// Copyright 2022 Confluent Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Refer to LICENSE for more information.
+
+using System;
+using System.Collections.Generic;
+
+
+namespace Confluent.Kafka.Admin
+{
+    /// <summary>
+    ///    Result for a delete ACLs operation with a list of <see cref="AclBinding" />
+    /// </summary>
+    public class DeleteAclsResult
+    {
+        /// <summary>
+        ///     List of ACL bindings in this result
+        /// </summary>
+        public List<AclBinding> AclBindings { get; set; }
```

Comment:
In the other admin operations, this is not a list here, rather a list is returned from the DeleteAclsAsync method.
We could argue about which way is better (i think the way the other methods are), but regardless, i think consistency wins out given what we have.

Reply:
ahh, i understand now why you want a list here. you want to return a list of lists.

Reply:
That is like this because each filter returns a different list of ACL bindings that have been deleted